.travis.yml

```diff
@@ -1,6 +1,8 @@
 # Test build the MoveIt! tutorials. Author: Dave Coleman
 sudo: required
-dist: xenial
+dist: trusty
+services:
+  - docker
```

Comment:
Good catch! I missed adding docker. Interestingly, it worked without: https://travis-ci.org/ros-planning/moveit_tutorials/builds/509295999